src/test/java/org/reaktivity/specification/amqp/streams/LinkIT.java

```diff
@@ -72,4 +72,16 @@ public void shouldExchangeAttachAsSenderAndReceiver() throws Exception
         k3po.notifyBarrier("ROUTED_SERVER");
         k3po.finish();
     }
+
+    @Test
+    @Specification({
+        "${scripts}/transfer.as.receiver/client",
+        "${scripts}/transfer.as.receiver/server"})
```

Comment:
Suggest renaming `transfer.as.receiver` to `transfer.to.client`.
Also rename method to `shouldTransferToClient`